### PR TITLE
Return stream

### PR DESCRIPTION
**Pull request by MaxDeg**
Enable api methods to return the body stream of the response directly (useful in case of file download). 
related to #218

### CODE REVIEW

File: Refit/RequestBuilderImplementation.cs [hunk 1]
```diff
@@ -330,7 +330,11 @@ void AddMultipartItem(MultipartFormDataContent multiPartContent, string fileName
         Func<HttpClient, object[], Task> BuildVoidTaskFuncForMethod(RestMethodInfo restMethod)
         {                      
             return async (client, paramList) => {
+<<<<<<< HEAD
                 var factory = buildRequestFactoryForMethod(restMethod.Name, client.BaseAddress?.AbsolutePath, restMethod.CancellationToken != null);
+=======
```
Comment (WilkaH) [inline Refit/RequestBuilderImplementation.cs]: Looks like there's some unresolved merge conflicts here.

File: Refit/RequestBuilderImplementation.cs [hunk 2]
```diff
@@ -382,6 +382,11 @@ void AddMultipartItem(MultipartFormDataContent multiPartContent, string fileName
                 if (restMethod.SerializedReturnType == typeof(HttpContent)) {
                     return (T)(object)resp.Content;
                 }
+
+                if (restMethod.SerializedReturnType == typeof(Stream))
+                {
```
Comment (Cheesebaron) [inline Refit/RequestBuilderImplementation.cs]: Use same style as if above

Comment (MaxDeg) [inline Refit/RequestBuilderImplementation.cs]: done ;)

**Comment by onovotny** [comment]
Please remove `Upgradelog.htm` and then :shipit:

**Comment by MaxDeg** [comment]
Btw any idea about when a new version of Refit will be available? Is it plan for 4.0.0?
Thanks ;)

**Pull request merged**